dspace/config/modules/usage-statistics.cfg

```diff
@@ -28,6 +28,8 @@ usage-statistics.authorization.admin.usage=true
 usage-statistics.authorization.admin.search=true
 #Workflow result statistics
 usage-statistics.authorization.admin.workflow=true
+#Number of results to show in statistics-home for usage stats
+itemcount.usage = 10
```

Comment:
Needs to be renamed `usage-statistics.itemcount.usage` to support the new DSpace 6 Configuration System: https://wiki.duraspace.org/display/DSDOC6x/Configuration+Reference